docs/developer-walk-through.rst

```diff
@@ -1,140 +1,14 @@
-Developer Walk-through
-======================
+Other stuff
+===========
 
 .. toctree::
    :maxdepth: 2
    :caption: Contents
 
-   main-developer-workflow
-   how-to-add-a-new-script-to-the-cea
    the-configuration-file
    configuration-file-details
-   how-to-create-a-new-release
```

Comment:
where is this going? we kind of still need this information, but totally ok with moving it around.

Reply:
yes it is kept already but re-organized